express_relay/sdk/python/express_relay/types.py

```diff
@@ -193,6 +193,21 @@ class OpportunityParams(BaseModel):
     params: Union[OpportunityParamsV1] = Field(..., discriminator="version")
 
 
+class EIP712Domain(BaseModel):
+    """
+    Attributes:
+        name: The domain name parameter for the EIP712 domain separator.
```

Comment:
nit: these comments look a bit repetitive and information-less

Reply:
I just removed the comments